model/src/main/scala/org/broadinstitute/dsde/rawls/RawlsException.scala

```diff
@@ -1,11 +1,25 @@
 package org.broadinstitute.dsde.rawls
 
-import org.broadinstitute.dsde.rawls.model.ErrorReport
+import akka.http.scaladsl.model.StatusCode
+import org.broadinstitute.dsde.rawls.model.{ErrorReport, ErrorReportSource}
 
 class RawlsException(message: String = null, cause: Throwable = null) extends Exception(message, cause)
 
 class RawlsExceptionWithErrorReport(val errorReport: ErrorReport) extends RawlsException(errorReport.toString)
 
+object RawlsExceptionWithErrorReport {
```

Comment:
just adding some quality of life improvements, so we don't need to explicitly create an error report every time